### DISCUSSION TITLE
Rust call a callback in a function

### DISCUSSION THREAD

**Discussion by ExoticDG** [slint-ui/slint#5626] [Q&A] [answered]
Hello. I was wondering if it was possible to call a callback in a function that is made in the appwindow.slint file. 
For example:

```
export component AppWindow inherits Window {

property <string> Answer: "";
    property <string> Problem: "";
    callback calculate_num1(string);
    callback calculate_num2(string);
    callback calculate_sym(string);

   function calculate () {
      // call callback here
   }

}
```

If that is not possible is there any other way to call multiple callbacks with one press of a button?

**Comment by tronical** [comment]
You can call calculate_num1, calculate_num2, and callback_sym from calculate(). Callbacks are like functions, except that you can change their “destination” at runtime.

**Reply by ExoticDG** [reply]
Ok thanks. Could you provide an example of how to call a callback in a function? I'm new to slint and am not exactly sure how to do that.

**Reply by tronical** [reply, marked as answer]
A callback is called as if it were a function: `calculate_num(“1234”)` for example

**Reply by ExoticDG** [reply]
Thank you.
